docs/guide/execute-transaction.md

```diff
@@ -79,11 +79,11 @@ Examples of parameter [`ExecParams`](https://algobuilder.dev/api/algob/modules/r
   }
 ```
 
-### [Deploy SSC](https://algobuilder.dev/api/algob/modules/runtime.types.html#deploysscparam)
+### [Deploy SSC](https://algobuilder.dev/api/algob/modules/runtime.types.html#deployAppParam)
```

Comment:
If we did a search & replace accross all files, then we need to update links here. i.e it should be `https://algobuilder.dev/api/algob/modules/runtime.types.html#deployappparam`

Reply:
no, it doesn't work like that, had to replace word by word , because types has capital letters, functions don't similarly for others